src/backends/backend.js

```diff
@@ -96,6 +97,13 @@ class Backend {
         {
           entries: entries.map(this.entryWithFormat(collection)),
         }
+      ))
+      .then(loadedCollection => (
```

Comment:
We're trying to get better about source docs - can you add a super brief comment explaining this new link in the promise chain?